pkg/log/loglevels.go

```diff
@@ -0,0 +1,22 @@
+package log
+
+// Log levels defined for use with:
+//   klog.V(__).Info
+
+const (
+	// INFO_LEVEL : This level is for anything which does not happen very often, and while not being
+	//              an error, needs to be on the logs, because it's important information:
+	//                * startup information
+	//                * HA failovers
+	//                * re-connections/disconnections
+	//                * ...
+	//              This level is not defined because you must use klog.Info helpers
+	//
+	// DEBUG_LEVEL : should be used to provide logs on often occurring events but, which could be helpful
+	//               for debugging errors.
+	DEBUG_LEVEL = 2
```

Comment:
Do you think we can omit the "_LEVEL" part to reduce real estate? I think `klog.V(log.DEBUG).Infof(...)` reads clearly.

Reply:
Big +1 to this.

Reply:
I like that, will do it.